src/overlays/actors/ovl_En_Lift_Nuts/z_en_lift_nuts.c

```diff
@@ -775,40 +775,40 @@ void EnLiftNuts_Move(EnLiftNuts* this, PlayState* play) {
 }
 
 void EnLiftNuts_SetupMovePlayer(EnLiftNuts* this) {
-    this->actionFunc = EnLiftNuts_MovePlayer;
+    this->actionFunc = EnLiftNuts_MovePlayerToPos;
 }
 
-void EnLiftNuts_MovePlayer(EnLiftNuts* this, PlayState* play) {
+void EnLiftNuts_MovePlayerToPos(EnLiftNuts* this, PlayState* play) {
```

Comment:
I don't think this function should be called `MovePlayerToPos` since it doesn't get a position passed as argument.
What about `MovePlayerToActor`/`MovePlayerToSelf`?

Reply:
I think `EnLiftNuts_MovePlayerToHomePos` would be fine?